frontend/amundsen_application/static/js/config/config-default.ts

```diff
@@ -284,6 +284,9 @@ const configDefault: AppConfig = {
           iconPath: '/static/images/github.png',
         },
       },
+      stats: {
+        statTypesToExcludeForIcon: [],
```

Comment:
`iconNotRequiredTypes` ? or something like that makes the intended behavior clearer I think